package.json

```diff
@@ -11,6 +11,10 @@
     "build": "yarn workspaces foreach --all --topological --verbose run build",
     "publint": "yarn workspaces foreach --all --topological --verbose run publint",
     "publish": "yarn workspaces foreach --all --no-private --topological --verbose npm publish --tolerate-republish && changeset tag",
+    "version:npm": "changeset version",
+    "version:jsr": "yarn workspaces foreach --all --no-private --parallel run version:jsr",
+    "version:get": "jq -r .version $INIT_CWD/package.json",
+    "version:set": "jq --arg version $0 '.version = $version' $INIT_CWD/deno.json | sponge $INIT_CWD/deno.json",
```

Comment:
Uses `jq` to update the `version` in the workspace `deno.json`, `$0` comes from `yarn run version:get` and `$INIT_CWD` is the current workspace directory. `sponge` waits for all output from `jq` before opening and writing to the output file